src/main/java/com/GeorgesServer/app/Router.java

```diff
@@ -26,6 +23,9 @@ public IHandler route(String publicFolderPath, ClientRequest request) {
         if (request.getMethod().equals("OPTIONS")) {
             return new OptionsHandler(request);
         }
+        if (request.getUrl().equals("/foobar") && !request.getMethod().isEmpty()) {
```

Comment:
This method is starting to get a little crazy.

Reply:
Even though cobspec only has 404 for the "/foobar" path it's more helpful to think of it as a catchall for any request that isn't handled by another handler.